.github/workflows/scripts/linux_arm64/rh8/build_catalyst.sh

```diff
@@ -79,6 +79,7 @@ cmake --build quantum-build --target check-dialects compiler_driver catalyst-cli
 
 # Copy files needed for the wheel where they are expected
 cp /catalyst/runtime-build/lib/*/*/*/*/librtd* /catalyst/runtime-build/lib
+find . -name 'libcatalyst_python_interpreter.*' -exec cp {} /catalyst/runtime-build/lib \;
```

Comment:
Any reason why we need to run `find` here, rather than specifying the path directly in a `cp` command?

Reply:
Couldn't we copy all shared libraries in `/catalyst/runtime-build/lib` instead?